src/pages/task/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useBreadcrumbAnalytics } from "analytics";
```

Comment:
should the breadcrumbs folder name be lowercase?

<img width="186" alt="Screen Shot 2022-10-11 at 9 16 03 AM" src="https://user-images.githubusercontent.com/47064971/195101334-9163701f-b4fe-4db9-a947-3468796463cc.png">

Reply:
I wish we went with the same pattern we use for the components folder capital folder names and all the logic in the folder. :/

 But i guess i can lower case it now for consistency but we WILL change it in the future.

Reply:
do we have a ticket for this? if not, would you mind writing a ticket 🙏

Reply:
https://jira.mongodb.org/browse/EVG-18051 Just opened!